parser/utils.go

```diff
@@ -49,3 +49,10 @@ func printHelp(argv []string, usage string) bool {
 
 	return false
 }
+
+func addGlobalFlags(input string) string {
```

Comment:
global client flags don't have to be part of every help doc. It just needs to be part of the main deis usage as seen in deis.go. Why do we need it in every command?